Demo/src/main/java/com/braintreepayments/demo/PayPalFragment.java

```diff
@@ -51,6 +52,13 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
             launchPayPal(false, buyerEmailEditText.getText().toString());
         });
         billingAgreementButton.setOnClickListener(v -> {
+            FragmentActivity activity = getActivity();
+
+            if (Settings.isPayPalAppSwithEnabled(activity) && buyerEmailEditText.getText().toString().isEmpty()) {
+                Toast.makeText(activity, "Email cannot be nil for App Switch flow", Toast.LENGTH_SHORT).show();
```

Comment:
```suggestion
                Toast.makeText(activity, "Email is required for the App Switch flow", Toast.LENGTH_SHORT).show();
```

Nit on the toast wording.